test/index.js

```diff
@@ -66,85 +66,101 @@ describe('<ReactQuill />', function() {
     expect(quill.options.readOnly).to.equal(props.readOnly);
     expect(quill.options.modules).to.include.keys(Object.keys(props.modules));
     expect(quill.options.formats).to.include.members(props.formats);
-  })
+  });
 
   it('allows using HTML strings as value', () => {
     const html = '<p>Hello, world!</p>';
-    const wrapper = mountReactQuill({value: html});
+    const wrapper = mountReactQuill({ value: html });
     const quill = getQuillInstance(wrapper);
     expect(getQuillContentsAsHTML(wrapper)).to.equal(html);
   });
 
   it('allows using HTML strings as defaultValue', () => {
     const html = '<p>Hello, world!</p>';
-    const wrapper = mountReactQuill({defaultValue: html});
+    const wrapper = mountReactQuill({ defaultValue: html });
     const quill = getQuillInstance(wrapper);
     expect(getQuillContentsAsHTML(wrapper)).to.equal(html);
   });
 
   it('allows using Deltas as value', () => {
     const html = '<p>Hello, world!</p>';
-    const delta = {ops: [{insert: 'Hello, world!'}]};
-    const wrapper = mountReactQuill({value: html});
+    const delta = { ops: [{ insert: 'Hello, world!' }] };
+    const wrapper = mountReactQuill({ value: html });
     const quill = getQuillInstance(wrapper);
     expect(getQuillContentsAsHTML(wrapper)).to.equal(html);
   });
 
-  it('prevents using Delta changesets from events as value', (done) => {
+  it('prevents using Delta changesets from events as value', done => {
     const value = '<p>Hello, world!</p>';
     const changedValue = '<p>Adieu, world!</p>';
+    let wrapper;
     const onChange = (_, delta) => {
-      withMockedConsole(() => {
-        expect(() => wrapper.setProps({value: delta})).to.throw();
-        done();
-      });
+      wrapper.setProps({ value: delta });
     };
-    const wrapper = mountReactQuill({value, onChange});
+    wrapper = mountReactQuill({ value, onChange });
+
+    const expectedErr = /You are passing the `delta` object from the `onChange` event back/;
+    // this test knows a lot about the implementation,
+    // but we need to wrap the right function with a catch
+    // in order to prevent errors from it from propagating
+    const origValidateProps = wrapper.instance().validateProps;
+    let calledDone = false; // might get called more than once
+    wrapper.instance().validateProps = function(props) {
+      try {
```

Comment:
Changed this to a full try/catch because this is the only test that didn't work when running in the Cypress browser instead of JSDOM; it seems to not fully stop the error that is indirectly triggered by `validateProps`:

```diff
- expect(() => wrapper.setProps({value: delta})).to.throw();
```